client/lxd_instances.go

```diff
@@ -484,6 +484,48 @@ func (r *ProtocolLXD) GetInstance(name string) (*api.Instance, string, error) {
 	return &instance, etag, nil
 }
 
+// GetInstanceEFIVars returns the instance EFI variables list for the provided name.
+func (r *ProtocolLXD) GetInstanceEFIVars(name string) (*api.InstanceEFI, string, error) {
+	instanceEFI := api.InstanceEFI{}
+
+	path, _, err := r.instanceTypeToPath(api.InstanceTypeAny)
+	if err != nil {
+		return nil, "", err
+	}
+
+	if r.CheckExtension("instances_efi_vars") != nil {
+		return nil, "", fmt.Errorf("EFI variables manipulation is not supported by server")
```

Comment:
You can just use this pattern here, as the error returned includes a message saying the extension is missing from the server:

```go
	err := r.CheckExtension("instances_efi_vars")
	if err != nil {
		return nil, err
	}
```